portal-ui/src/screens/Console/Tenants/TenantDetails/TenantCSR.tsx

```diff
@@ -0,0 +1,117 @@
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { Theme } from "@mui/material/styles";
+import createStyles from "@mui/styles/createStyles";
+import withStyles from "@mui/styles/withStyles";
+import {
+  actionsTray,
+  containerForHeader,
+  searchField, tableStyles,
+} from "../../Common/FormComponents/common/styleLibrary";
+import Grid from "@mui/material/Grid";
+import { setErrorSnackMessage } from "../../../../actions";
+import { ErrorResponseHandler } from "../../../../common/types";
+import api from "../../../../common/api";
+import { AppState } from "../../../../store";
+import LabelValuePair from "../../Common/UsageBarWrapper/LabelValuePair";
+
+interface ITenantCSRProps {
+  classes: any;
+  match: any;
+  loadingTenant: boolean;
+  setErrorSnackMessage: typeof setErrorSnackMessage;
+}
+
+const styles = (theme: Theme) =>
+    createStyles({
+      tableWrapper: {
+        height: "450px",
+      },
+      ...actionsTray,
+      ...searchField,
+      ...tableStyles,
+      ...containerForHeader(theme.spacing(4)),
+    });
+
+const TenantCSR = ({
+                        classes,
+                        match,
+                        loadingTenant,
+                        setErrorSnackMessage,
+                      }: ITenantCSRProps) => {
+  const [loading, setLoading] = useState<boolean>(true);
+  const tenantName = match.params["tenantName"];
+  const tenantNamespace = match.params["tenantNamespace"];
+  const [status, setStatus] = useState("");
+  const [name, setName] = useState("");
+  const [annotations, setAnnotations] = useState("");
+  useEffect(() => {
+    if (loadingTenant) {
+      setLoading(true);
+    }
+  }, [loadingTenant]);
+
+  useEffect(() => {
+    if (loading) {
+      api
+          .invoke(
+              "GET",
+              `/api/v1/namespaces/${tenantNamespace}/tenants/${tenantName}/csr`
+          )
+          .then((res) => {
+              setStatus((status) => res.status);
+              setName((name) => res.name);
+              setAnnotations((annotations) => res.annotations)
+              setLoading(false);
+          })
+          .catch((err: ErrorResponseHandler) => {
+            setErrorSnackMessage(err);
+            setLoading(false);
+          });
+    }
+  }, [loading, tenantNamespace, tenantName, setErrorSnackMessage]);
+
+  return (
+      <React.Fragment>
```

Comment:
Please just call <Fragment>
```suggestion
      <Fragment>
```